src/components/backend-ai-summary-view.ts

```diff
@@ -414,15 +416,17 @@ export default class BackendAISummary extends BackendAIPage {
       })
   }
 
-  _deleteInvitation(invitation: any) {
+  _deleteInvitation(e, invitation: any) {
     if (!this.activeConnected) {
       return;
     }
     globalThis.backendaiclient.vfolder.delete_invitation(invitation.id)
       .then(res => {
+        let panel = e.target.closest('lablup-activity-panel');
```

Comment:
Same here.

Reply:
Fixed. Please double-check the fixed code.